lib/datasource/pod/index.ts

```diff
@@ -11,6 +11,9 @@ export const defaultRegistryUrls = ['https://cdn.cocoapods.org'];
 const cacheNamespace = `datasource-${id}`;
 const cacheMinutes = 30;
 
+const githubHttp = new GithubHttp();
+const http = new GithubHttp();
```

Comment:
Why is this `GitHubHttp` instead of `Http`?

Reply:
Oops